src/video/video.js

```diff
@@ -136,7 +136,7 @@
             // sanitize potential given parameters
             settings.doubleBuffering = !!(settings.doubleBuffering);
             settings.autoScale = (settings.scale === "auto") || false;
-            settings.maintainAspectRatio = !!(settings.maintainAspectRatio);
+            settings.scaleMethod = !!(settings.scaleMethod);
```

Comment:
This transforms the value into a boolean. I don't think you want to do that with a string input.